e2e-tests/gef/cypress/integration/about-facility.spec.js

```diff
@@ -1,12 +1,15 @@
 /* eslint-disable no-underscore-dangle */
 /* eslint-disable no-undef */
+import moment from 'moment';
 import relative from './relativeURL';
 import aboutFacility from './pages/about-facility';
 import CREDENTIALS from '../fixtures/credentials.json';
 
 const applications = [];
 let token;
 
+const now = moment();
```

Comment:
should we be using date-fns for this?

Reply:
Im leaving the cypress piece for now as it uses moment everywhere